code/API_definitions/population-density-data.yaml

```diff
@@ -595,11 +689,11 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            $ref: '#/components/schemas/ErrorInfo'
           example:
             status: 400
-            code: "INVALID_ARGUMENT"
-            message: "Client specified an invalid argument, request body or query param"
+            code: 'INVALID_ARGUMENT'
+            message: 'Client specified an invalid argument, request body or query param'

```

Comment:
What about adding also to be fully aligned with commonalities for the subscription?

```
           GENERIC_400_INVALID_PROTOCOL:
              value:
                status: 400
                code: "INVALID_PROTOCOL"
                message: "Only HTTP is supported"
            GENERIC_400_INVALID_CREDENTIAL:
              value:
                status: 400
                code: "INVALID_CREDENTIAL"
                message: "Only Access token is supported"
            GENERIC_400_INVALID_TOKEN:
              value:
                status: 400
                code: "INVALID_TOKEN"
                message: "Only bearer token is supported"`

Reply:
hope it's solved now, added (except Invalid_protocol, not sure where it comes from) and also added generic missing 410